src/components/common/ColonyHome/ColonyNavigation/NavItem.tsx

```diff
@@ -1,61 +1,72 @@
-import React, { KeyboardEvent, useCallback } from 'react';
-import { MessageDescriptor, useIntl } from 'react-intl';
+import React, { KeyboardEvent } from 'react';
+import { MessageDescriptor } from 'react-intl';
+import { useLocation } from 'react-router-dom';
 
 import NavLink from '~shared/NavLink';
 import { ENTER } from '~types/index';
+import { formatText } from '~utils/intl';
 
 import styles from './NavItem.css';
 
-interface Props {
+export interface NavItemProps {
   disabled?: boolean;
   // exact?: boolean;
-  extra?: MessageDescriptor;
   linkTo: string;
   showDot?: boolean;
   text: MessageDescriptor;
   dataTest?: string;
 }
 
-const displayName = 'dashboard.ColonyHome.ColonyNavigation.NavItem';
+const displayName = 'common.ColonyHome.ColonyNavigation.NavItem';
+
+const handleLinkKeyDown = (
+  evt: KeyboardEvent<HTMLAnchorElement>,
```

Comment:
For future reference, should we stick with evt, event or e as a variable name for events?

Reply:
If I may butt in, I'm a fan of `e`. Simple, elegant, and I think everyone understands by now that an `e` is short for `event`